packages/ag-charts-website/src/content/docs/sunburst-series/_examples/org-chart/data.ts

```diff
@@ -0,0 +1,101 @@
+let rowData: any[] = [
```

Comment:
This is duplicated from the treemap - maybe there's a better way

Reply:
This is ok, we don't share code between examples so they are as standalone as possible.